sys-power/suspend/files/suspend-1.0_p20200924-Use-pkgconf-for-libgcrypt.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/configure.ac b/configure.ac
+index 5f3adb5..b7f21eb 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -211,7 +211,12 @@ fi
+ if test "${enable_encrypt}" = "yes"; then
+ 	CONFIG_FEATURES="${CONFIG_FEATURES} encrypt"
+ 	AC_DEFINE([CONFIG_ENCRYPT], [1], [Define if encryption enabled])
+-	AM_PATH_LIBGCRYPT([1:1.6.3], [], [AC_MSG_ERROR([Cannot locate >=libgcrypt-1.6.3])])
++	PKG_CONFIG="pkg-config --static"
```

Comment:
This looks a bit off to me. @SoapGentoo?

Reply:
This particular construction is used elsewhere in file by the author, because PKG_CHECK_MODULES_STATIC did not exist when this program was written.

Reply:
OK.